test/extended/testdata/test-imagesource-build.yaml

```diff
@@ -89,7 +89,7 @@ items:
           deploymentconfig: imagesourceapp
       spec:
         containers:
-        - image: imagesourceapp
+        - image: " "
```

Comment:
To answer the earlier question, we are only doing validation fix up on empty string; single space passes the k8s validation